server/src/main/resources/org/eclipse/lsp/cobol/service/delegates/completions/LanguageKeywords.txt

```diff
@@ -2019,16 +2019,92 @@ SQLADM=
 SQLC=
 SQLCCSID=
 SQLCA=
+SQLCA-AUTHID=
```

Comment:
In this case, they will be shown as keywords while effectively they are variables.

Reply:
> will be s

As I wrote, we have second-part, concentrating on it.